src/docs/quick-start.rst

```diff
@@ -1,12 +1,201 @@
+
+.. code-block:: text
+
+    realm join sssd.io
+
+It will default and use the *Administrator* user, add the -u flag to specify a different user account to join the domain.
+
+.. code-block:: text
+
+    realm join -u jsmith sssd.io
+
+Now see if it works, and issue an id command.
+
+.. code-block:: text
+
+    id administrator@sssd.io
+
+If you want to use short names, edit ``sssd.conf`` and set ``use_fully_qualified_names`` to ``false``.
+
+.. note::
+
+    In the event of user name conflict, jsmith@sssd.io, jsmith@child.sssd.io for example, you can configure a domain resolution order using shortnames. :doc:`short names <../design-pages/pages/shortnames>`. If that does not work, checkout ``sss_overide`` which is part of the ``sssd_tools`` package to create a local override. Of course it's best to resolve the conflict.
+
+.. code-block:: text
+
+    id administrator
+
+The following command and logins should now work. For more detail please refer to :doc:ad/ad-provider .
+
+Quick Start LDAP
+****************
+
+Before starting make sure you have the following information:
+
+- LDAP domain *i.e. sssd.io*
+- LDAP suffix *i.e. DC=sssd,DC=io*
+- LDAP bind user *i.e. UID=bind_user,OU=people,DC=sssd,DC=io*
+- LDAP bind password *i.e. password123*
+- LDAP server hostname *i.e. ldap1.sssd.io*
+- LDAP server IP *i.e. 1.2.3.4*
+- LDAP server CA certificate *i.e. /etc/openldap/cacerts/ca.crt*
+
+.. note::
+
+    The bind user and the bind password are only necessary if the LDAP server you are connecting does not permit anonymous binds.
+
+First install the necessary package, sssd.
+
+.. code-tabs::
+
+    .. fedora-tab::
+
+        dnf install -y sssd
+
+    .. rhel-tab::
+
+        yum install -y sssd
+
+Edit ``/etc/sssd/sssd.conf`` and add a new domain section. The section should look like the following without a bind user.
+
+.. code-block:: ini
+   :caption: /etc/sssd/sssd.conf
+
+    [sssd]
+    domains = LDAP_DOMAIN
+
+    [domain/LDAP_DOMAIN]
+    id_provider = ldap
+    auth_provider = ldap
+
+    ldap_uri = ldap://ldap1.sssd.io
+    ldap_search_base = DC=sssd,DC=io
+
+    ldap_id_use_start_tls = true
+    ldap_tls_reqcert = demand
+    ldap_tls_cacert = /etc/openldap/cacerts/ca.crt
+
+.. note::
+
+    CA certificates are usually kept in ``/etc/openldap/cacert``, and ``start_tls`` or ``ldaps`` should be used, **DO NOT** use both at the same time.
+
+If anonymous queries are not enabled on the server, the following section is required for the bind account.
+
+.. code-block:: text
+
+    ldap_default_bind_dn = uid=bind_user,ou=people,dc=sssd,dc=io
+    ldap_default_authtok_type = password
+    ldap_default_authtok = password123
+
+In RHEL8 and clones you should use ``authselect`` to configure the rest of the configuration, on older versions like RHEL7 use ``authconfig``.
+
+.. code-tabs::
+
+    .. fedora-tab::
+
+        authselect select sssd
+
+    .. rhel-tab::
+        :version: 8+
```

Comment:
In other tabs on the page we refer `Fedora` or `RHEL`. Here we explicit mention RHEL version. Maybe for consistency there should be one RHEL tab here and in this tab info about RHEL7 / RHEL8 command?

Reply:
This is fine and intended usage of :versions: